mbuild/tests/test_coordinate_transform.py

```diff
@@ -346,6 +346,19 @@ def test_rotate_around_z_away_from_origin(self, sixpoints):
             before[:, 1], -1 * after[:, 1], atol=1e-16
         ) and np.allclose(before[:, 0], -1 * after[:, 0], atol=1e-16)
 
+    def test_rotate_dihedral(self, ethane):
```

Comment:
I think we should add a test with a less symmetric molecule than ethane. This is a better proof of concept that the atoms were translated properly, especially if we're dealing with a long molecule polymers.